README.md

```diff
@@ -320,6 +320,22 @@ For more information, please head [here](https://github.com/dotnet/docs/blob/mas
 ```bash
     docker push registry.heroku.com/<heroku-app-name>/web
 ```
+### Rollbar
+
+Rollbar is a tool that allows monitoring errors from your application.
+
+To use Rollbar, you only need to put the access token provided by rollbar in the appsettings.Development.json file.
```

Comment:
To use Rollbar, you only need to put the access token provided by rollbar in the appsettings file.